plugins/flytekit-inference/flytekitplugins/inference/ollama/serve.py

```diff
@@ -36,7 +55,10 @@ def __init__(
     ):
         """Initialize Ollama class for managing a Kubernetes pod template.
 
-        :param model: An instance of the Model class containing the model's configuration, including its name, memory, CPU, and file.
+        Python 3.12 or higher is required due to support for backslashes in f-strings:
+        https://realpython.com/python312-f-strings/#backslashes-now-allowed-in-f-strings
```

Comment:
I think using the `NEWLINE` and `NEWLINE_ESCAPED` global vars should address this